vmaas_sync/vmaas_sync.py

```diff
@@ -198,6 +205,51 @@ def delete_cve(cve_names: str) -> bool:
     return success
 
 
+def sync_exploits():
```

Comment:
I vote to move this code to taskomatic job where it can be run periodically. It doesn't make sense to be attached to vmaas sync websocket event.

Reply:
It could be, but in a case when we get new CVEs from vmaas there might be CVEs with wrong exploit status until the taskomatic sync exploits

Reply:
OK, sounds reasonable. Althought it doesn't handle the case when exploit is added between 2 vmaas synces.